### ISSUE TITLE
API Tracking issue: groups.read

### ISSUE THREAD

**Issue by bibliotechie** [bibliotechie/antithesis#16]
This is a tracking issue for the Hypothesis API endpoint `GET /groups`. Corresponding wiki page: https://github.com/bibliotechie/antithesis/wiki/Hypothesis-API%3A-groups.read

**Note:** This is **not** the tracking issue for `group.read`; that can be found at #17.

**Comment by bibliotechie** [comment]
I'm going to break out the portion we'll probably need for MVP into a separate issue, in case we decide to implement the full groups API later.

**Comment by bibliotechie** [comment]
Depends on #33.

**Comment by bibliotechie** [comment]
> Depends on #33.

(It doesn't really, but we'll track it as such.)
